src/backend/base/langflow/initial_setup/setup.py

```diff
@@ -615,3 +616,17 @@ def initialize_super_user_if_needed():
         create_default_folder_if_it_doesnt_exist(session, super_user.id)
         session.commit()
         logger.info("Super user initialized")
+
+
+# Function to download NLTK packages if not already downloaded
+def download_nltk_resources():
+    nltk_resources = {"tokenizers": ["punkt", "punkt_tab"], "taggers": ["averaged_perceptron_tagger"]}
+
+    for category, packages in nltk_resources.items():
+        for package in packages:
+            try:
+                nltk.data.find(f"{category}/{package}")
+                print(f"{package} ({category}) already exists.")
```

Comment:
is there a better logging function than just print() ?
I saw logging.info() is used 11  lines above